packages/@aws-cdk/aws-apigatewayv2/test/http/integrations/http.test.ts

```diff
@@ -38,4 +38,27 @@ describe('HttpProxyIntegration', () => {
       IntegrationMethod: 'PATCH',
     });
   });
-});
\ No newline at end of file
+
+  test('CORS Configuration is correctly configured.', () => {
```

Comment:
Can you add a test case that `CorsConfiguration` is `ABSENT` when not specified?